TTS/trainer.py

```diff
@@ -439,7 +455,7 @@ def _restore_list_objs(states, obj):
             if "scaler" in checkpoint and self.use_amp_scaler and checkpoint["scaler"]:
                 print(" > Restoring Scaler...")
                 scaler = _restore_list_objs(checkpoint["scaler"], scaler)
-        except (KeyError, RuntimeError):
+        except (KeyError, RuntimeError, ValueError):
```

Comment:
Because I found a case where an error occurred and returned ValueError and thus made the partial load of the model unfeasible. I don't remember what the case was anymore (maybe different shapes in the layers during restore?) but I find it necessary to always try to partial load as it is more efficient than training from scratch.